syndeo-template/src/main/java/com/google/cloud/syndeo/transforms/pubsub/SyndeoPubsubWriteSchemaTransformProvider.java

```diff
@@ -122,9 +159,14 @@ public abstract static class SyndeoPubsubWriteConfiguration {
         "The name of the topic to write data to. " + "Format: projects/${PROJECT}/topics/${TOPIC}")
     public abstract String getTopic();
 
-    public static SyndeoPubsubWriteConfiguration create(String format, String topic) {
+    public abstract @Nullable String getDlqType();
```

Comment:
same comment as the readSchemaTransform.

Reply:
Please refer the above comment

Reply:
lets add dlq configs as a top level param.